src/soc/starfive/jh7100/src/iopad.rs

```diff
@@ -149,7 +149,7 @@ pub const syscon_iopad_ctrl_register102_REG_ADDR: u32 = SYSCON_IOPAD_CTRL_BASE_A
 pub const syscon_iopad_ctrl_register103_REG_ADDR: u32 = SYSCON_IOPAD_CTRL_BASE_ADDR + 0x19C;
 pub const syscon_iopad_ctrl_register104_REG_ADDR: u32 = SYSCON_IOPAD_CTRL_BASE_ADDR + 0x1A0;
 
-pub fn _SET_SYSCON_REG_register0_SCFG_gpio_pad_ctrl_0(v: u32) -> () {
+pub fn _SET_SYSCON_REG_register0_SCFG_gpio_pad_ctrl_0(v: u32)  {
```

Comment:
I'm surprised that clippy is happy with two spaces between the close paren and the open curly brace. It seems better to have one space here.

Reply:
rustfmt was my friend.